### DISCUSSION TITLE
Suggestion to remove the exclamation mark in console

### DISCUSSION THREAD

**Discussion by sovtec** [EssentialsX/Essentials#5071] [Ideas and Feature Suggestions] [answered]
Suggestion to remove the exclamation mark in the console when getting this message

**[EssentialsDiscord] You added your bot to your server incorrectly. Please follow the tutorial in the config and add your bot using [https://essentialsx.net/discord.html!](https://essentialsx.net/discord.html!)** 

It may confuse someone that does not notice the exclamation mark at the end of the link and therefore thinks the link is broken

**Comment by JRoy** [comment, marked as answer]
Will be addressed in https://github.com/EssentialsX/Essentials/pull/5079
